apis/r/tests/testthat/test-11-shape.R

```diff
@@ -360,6 +360,21 @@ test_that("SOMADataFrame domain mods", {
         myint = c(20, 50),
         myfloat = c(0.0, 6.0)
     )
+
+    # -- first check dry run
+    expect_no_condition(sdf$change_domain(domain_for_create, check_only=TRUE))
+    sdf$close()
+
+    check <- list(
+        soma_joinid = c(0, 3),
+        mystring = c("", ""), # this is how it reads back
+        myint = c(20, 50),
+        myfloat = c(0.0, 6.0)
+    )
```

Comment:
Use the standard two-spaces for indentations
```suggestion
    check <- list(
      soma_joinid = c(0, 3),
      mystring = c("", ""), # this is how it reads back
      myint = c(20, 50),
      myfloat = c(0.0, 6.0)
    )
```